skyportal/handlers/base.py

```diff
@@ -0,0 +1,16 @@
+from baselayer.app.handlers.base import BaseHandler as BaselayerHandler
+import skyportal
```

Comment:
I'm worried a bit about circular dependencies here.  Maybe `from ..version import __version__`?

Reply:
Hmm, that doesn't work - there's no `version` module - are you suggesting we add it?

Reply:
@stefanv - changed